### ISSUE TITLE
Deselecting accessories in workout remove all sets

### ISSUE THREAD

**Issue by aaron25mt** [JonnyBeeGod/nSuns-5-3-1-iOS-issues#33]
I've found that deselecting accessories for a workout will remove it even if it has completed sets. For example, if I select all my accessories and insert all my sets but I go to add another exercise and accidentally deselect it, it'll remove all completed sets. This occurs when you deselect the exercise and press Done and even if you deselect the exercise, then reselect it, then press Done.

**Comment by JonnyBeeGod** [comment]
I agree, terrible UX. Deselecting an exercise in the Assistance Exercise Catalog should not be possible if there are already recorded sets in this workout. Also I'd like to show in the catalog whether there are already recorded sets in this workout
